src/main/java/de/tum/cit/aet/artemis/core/web/FileResource.java

```diff
@@ -498,9 +498,9 @@ public ResponseEntity<byte[]> getLecturePdfAttachmentsMerged(@PathVariable Long
      * @return The requested file, 403 if the logged-in user is not allowed to access it, or 404 if the file doesn't exist
      */
     @GetMapping("files/attachments/attachment-unit/{attachmentUnitId}/*")
-    @EnforceAtLeastStudent
+    @EnforceAtLeastTutor
```

Comment:
Not quite sure about this: Students will not be able to see attachment-units anymore, is this intended?

In particular wondering as there is a check for hidden, but can't teaching assistants always see the slide?